### PR TITLE
=doc hide auto downing from default sample config in cluster

### PR DESCRIPTION
**Pull request by ktoso**
I checked how it looks in generated docs – looks OK.

### CODE REVIEW

File: akka-samples/akka-sample-cluster-java/src/main/resources/application.conf [hunk 1]
```diff
@@ -15,7 +16,14 @@ akka {
       "akka.tcp://ClusterSystem@127.0.0.1:2551",
       "akka.tcp://ClusterSystem@127.0.0.1:2552"]
 
+    //#snippet
+    // excluded from snippet
     auto-down-unreachable-after = 10s
+    //#snippet
+    # auto downing is NOT safe for production deployments.
+    # you may want to use it during development, read more about it in the docs.
```
Comment (johanandren) [inline akka-samples/akka-sample-cluster-java/src/main/resources/application.conf]: :+1: :+1: :+1:

**Comment by johanandren** [comment]
LGTM

File: akka-samples/akka-sample-cluster-java/src/main/resources/application.conf [hunk 2]
```diff
@@ -1,3 +1,4 @@
+//#snippet
```
Comment (patriknw) [inline akka-samples/akka-sample-cluster-java/src/main/resources/application.conf]: #//#

File: akka-samples/akka-sample-cluster-java/src/main/resources/application.conf [hunk 3]
```diff
@@ -15,7 +16,14 @@ akka {
       "akka.tcp://ClusterSystem@127.0.0.1:2551",
       "akka.tcp://ClusterSystem@127.0.0.1:2552"]
 
+    //#snippet
+    // excluded from snippet
```
Comment (patriknw) [inline akka-samples/akka-sample-cluster-java/src/main/resources/application.conf]: that is not a comment

File: akka-samples/akka-sample-cluster-java/src/main/resources/application.conf (hunk 1 above)
Comment (patriknw) [inline akka-samples/akka-sample-cluster-java/src/main/resources/application.conf]: :+1:

File: akka-samples/akka-sample-cluster-java/src/main/resources/application.conf (hunk 3 above)
Comment (ktoso) [inline akka-samples/akka-sample-cluster-java/src/main/resources/application.conf]: It is actually, but I'll better stick to the `#` syntax here indeed :) 
See: https://github.com/typesafehub/config#features-of-hocon

File: akka-samples/akka-sample-cluster-java/src/main/resources/application.conf (hunk 2 above)
Comment (ktoso) [inline akka-samples/akka-sample-cluster-java/src/main/resources/application.conf]: ok, will do

**Comment by ktoso** [comment]
Addressed comments about comment style

**Comment by patriknw** [comment]
LGTM (it was not about style, it was syntax error)

**Comment by ktoso** [comment]
Pretty sure `//` works in typesafe config, see here https://github.com/akka/akka/pull/19724#discussion_r52359064 what was I missing?

**Comment by patriknw** [comment]
I didn't know that. Then I'm sorry for the noise.

**Comment by ktoso** [comment]
No problem at all `#` style is more consistent with what we do everywhere anyway so thanks for noticing!

**Pull request merged**